content/en/05-use-case-tutorials/network-monitoring.md

```diff
@@ -0,0 +1,134 @@
+</PoiTooltip>
+
+<PoiTooltip top={60} left={87}>
+<strong>Enable Hierarchy View</strong>
+{' '}
+for a different view of the graph.
+</PoiTooltip>
+
+<svg title='' src='use-case-tutorials/net_flow_graph.png'/>
+:::
+
+> This script shows a mapping of all the outgoing connections for the pods in the specified namespace.
+
+> Grey hexagonal icons represent pods making network requests. Blue circles represent remote endpoint. If a request’s remote endpoint is within the cluster, then the IP address is resolved to a pod/service name.
+
+> Let's filter the graph to only show communication to the `pl-nats` pod.
+
+3. Select drop down arrow next to the `to_entity_filter` argument, type `pl-nats`, and press enter to re-run the script.
+
+> The graph should update to only show network traffic sent to the `pl-nats` pod. Pixie uses NATS as our messaging system.
+
+4. Clear the `to_entity_filter` value by selecting the drop-down arrow and pressing enter.
+
+5. Scroll down to the table below the graph. This table contains the same data that is used to construct the graph above.
+
+## Live DNS request map
+
+A global view of DNS requests made within a cluster can be used to:
+
+- Quickly determine which services are making DNS requests.
+- Get high-level latency and throughput information.
+- Observe imbalances of throughput between DNS servers.
+
+Let’s use the px/dns_flow_graph script to see a graph of DNS requests in the cluster:
+
+1. Select `px/dns_flow_graph` from the `script` drop-down menu.
+
+::: div image-xl relative
+<PoiTooltip top={14} left={28}>
+<strong>Enter a string</strong>
+{' '}
+to filter the DNS requests.
+</PoiTooltip>
+
+<PoiTooltip top={50} left={40}>
+<strong>Hover over an Edge</strong>
+{' '}
+to see  network transmission stats. Thicker lines indicate more traffic.
+</PoiTooltip>
+
+<PoiTooltip top={72} left={57}>
+<strong>Click table column titles</strong>
+{' '}
+to sort the column data.
+</PoiTooltip>
+
+<svg title='' src='use-case-tutorials/dns_flow_graph.png'/>
+:::
+
+> This script shows all of the DNS requests made in the cluster, with latency and throughput stats.
+
+> This cluster doesn't have anything deployed except Pixie, so all we see are the kube-dns pods communicating with the metadata server.
+
+2. Click on the `LATENCY_AVG` column title to sort the table data by average latency.
+
+## Live TCP drops, retransmits map
+
+TCP drops and retransmits can indicate network connectivity issues that may affect application performance.
+
+Let's use the `px/tcp_drops`script to see a global view of TCP drops across the cluster.
+
+1. Select `px/tcp_drops` from the script drop-down menu.
+
+> This script is different from other PxL scripts. The two previous scripts simply queried the Pixie platform for data that is being continuously collected (DNS events, HTTP events, etc). This script deploys a new short-lived data source, then queries the new data source after some time has passed.
```

Comment:
i would make this an info

Reply:
This script is a PxL mutation script. While the previous scripts query the Pixie platform for data, this script extends Pixie to collect new data.